robotiq_s_model_articulated_gazebo/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>robotiq_s_model_articulated_gazebo</name>
+  <version>1.0.0</version>
+  <description>Launch files for spawning the robotiq s-model articulated gripper</description>
+
+  <maintainer email="dash@clearpathrobotics.com">Devon Ash</maintainer>
+  <author email="dash@clearpathrobotics.com">Devon Ash</author>
+
+  <license>BSD</license>
```

Comment:
To confirm on this: robotiq_s_model_articulated_gazebo seems upon outside inspection to be a new package and doesn't look like it needs to follow the other packages by the letter of the law. However, if everything else related to this package is Apache 2.0, Clearpath is fine with licensing this as Apache 2.0 to keep things straightforward for future contributors.